test/specs/excluded_output.json

```
@@ -0,0 +1 @@
+[]
```

If a path submitted does not match the `include` section in the index configuration, it will be silently dropped and not generate any output.